python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-isis.j2

```diff
@@ -30,9 +30,6 @@
 {%     if router_isis.mpls_ldp_sync_default is arista.avd.defined(true) %}
 | MPLS LDP Sync Default | {{ router_isis.mpls_ldp_sync_default }} |
 {%     endif %}
-{%     if router_isis.timers.local_convergence.protected_prefixes is arista.avd.defined(true) %}
-| Local Convergence Delay (ms) | {{ router_isis.timers.local_convergence.delay | arista.avd.default(10000) }} |
```

Comment:
Maybe let's move `ms` after the value to look like the other

Reply:
Fixed